dockers/docker-orchagent/supervisord.conf

```diff
@@ -117,6 +117,14 @@ autorestart=false
 stdout_logfile=syslog
 stderr_logfile=syslog
 
+[program:nbrmgrd]
+command=/usr/bin/nbrmgrd
+priority=15
```

Comment:
Duplicate priority with restore_neighbors. This shouldn't cause any issues with the way we use supervisor, but I thought I'd bring it to your attention to see if it was intentional. The same applies to the docker-sonic-vs file below.

Reply:
It was not intentional, fixed the values. Thanks for pointing out!